Fix/perp mobile tpsl modal

Kahnchan opened the pull request:
<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Added a dedicated full-page mobile Set TP/SL modal for Perp positions, reachable from the positions list.
  * Mobile flow now opens a navigable modal showing asset, position size, entry/mark prices, and TPSL inputs.
  * In-modal close/back navigation for smoother mobile experience.
  * Desktop behavior unchanged — continues to use the existing Set TP/SL dialog.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Kahnchan committed:
feat: implement SetTpslModal for setting take profit and stop loss in Perp trading

- Added SetTpslModal component to handle take profit and stop loss settings.
- Integrated navigation for mobile modal presentation.
- Updated PerpPositionsList to open SetTpslModal with relevant parameters.
- Enhanced routing configuration to include MobileSetTpsl route.
- Refactored ISetTpslParams interface for better type safety.

Kahnchan committed:
Merge branch 'x' of https://github.com/OneKeyHQ/app-monorepo into fix/perp_mobile_tpsl_modal

revan-zhang commented:
### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/79054a58-da59-44ef-ae37-907e423231c5?refs=0ac317352f84b3f16a2115999209d0ce142f5087&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/79054a58-da59-44ef-ae37-907e423231c5/license?refs=0ac317352f84b3f16a2115999209d0ce142f5087&source=prChecksComment)

Kahnchan committed:
Merge remote-tracking branch 'origin' into fix/perp_mobile_tpsl_modal

originalix commented on packages/kit/src/views/Perp/components/OrderInfoPanel/SetTpslModal.tsx:
```diff
@@ -244,100 +252,102 @@ const SetTpslForm = memo(
+                })}
+              </SizableText>
+              <SizableText size="$bodyMdMedium">
+                {positionSize.toNumber()} {position.coin}
```
positionSize.toFixed(szDecimals)

Kahnchan committed:
refactor: adjust layout in SetTpslModal for improved button placement

- Removed unnecessary padding from YStack.
- Moved TradingGuardWrapper to ensure the confirm button is consistently displayed within the form structure.

Kahnchan committed:
fix: update position size display in SetTpslModal for better precision

- Changed position size display from `toNumber()` to `toFixed(szDecimals)` for improved formatting.

Kahnchan committed:
Merge branch 'x' into fix/perp_mobile_tpsl_modal

sidmorizon committed:
Merge branch 'x' into fix/perp_mobile_tpsl_modal
